translations/french.json

```
@@ -1435,8 +1435,9 @@
   "Please note that this instance does not support creating a new cold wallet for non-administrators. However, you can import one from other wallet software.": "Notez que cette instance ne permet pas de créer un nouveau cold wallet pour les non-administrateurs. Cependant, vous pouvez en importer un depuis un autre logiciel.",
   "Please provide a connection string": "Veuillez fournir une chaîne de connexion",
   "Please provide a destination": "Veuillez fournir une destination",
```

Biome (2.4.16)

[error] The key Please provide a destination was already declared.

(lint/suspicious/noDuplicateObjectKeys)